packages/workspace/src/workspace/state/state.ts

```diff
@@ -23,6 +23,12 @@ import { StateStaticFilesSource } from '../static_files/common'
 
 export type StateMetadataKey = 'version' | 'hash'
 
+export type updateStateElementsArgs = {
```

Comment:
type name conventions
```suggestion
export type UpdateStateElementsArgs = {
```